nixos/modules/profiles/installation-device.nix

```diff
@@ -55,13 +55,13 @@ with lib;
     services.mingetty.autologinUser = "nixos";
 
     # Some more help text.
-    services.mingetty.helpLine =
-      ''
-
-        The "nixos" and "root" account have empty passwords.  ${
-          optionalString config.services.xserver.enable
-            "Type `sudo systemctl start display-manager' to\nstart the graphical user interface."}
-      '';
+    services.mingetty.helpLine = ''
+      The "nixos" and "root" accounts have empty passwords.
+      sshd can be started manually via 'systemctl start sshd'
+      - remember to set a password, or add to ~/.ssh/authorized_keys first.
```

Comment:
How about
```
The "nixos" and "root" accounts have empty passwords.

Type `sudo systemctl start sshd` to the SSH daemon.
You then must set a password for either "root"  or "nixos" with `passwd` to be able to login.
```

That's inline to what's in the manual, but we don't have an entry there about `authorized_keys`.
Also it may become confusing as to which `authorized_keys` file to add to since it could either be `root`'s home `nixos`'s.